packages/wagmi/src/client.ts

```diff
@@ -342,13 +342,13 @@ export class Web3Modal extends Web3ModalScaffold {
 
     // Check if coinbase injected connector is present
     const coinbaseConnector = filteredConnectors.find(
-      c => c.id === CoreConstants.CONNECTOR_RDNS_MAP[coinbaseSDKId]
+      c => c.id === CoreConstants.CONNECTOR_RDNS_MAP[ConstantsUtil.COINBASE_CONNECTOR_ID]
     )
 
     filteredConnectors.forEach(({ id, name, type, icon }) => {
       // If coinbase injected connector is present, skip coinbase sdk connector.
-      const shouldSkip =
-        (coinbaseConnector && id === coinbaseSDKId) || ConstantsUtil.EMAIL_CONNECTOR_ID === id
+      const isCoinbaseRepeated = coinbaseConnector && id === coinbaseSDKId
+      const shouldSkip = isCoinbaseRepeated || ConstantsUtil.EMAIL_CONNECTOR_ID === id
```

Comment:
Is this only for Coinbase and MM, or can we make `isCoinbaseRepeated` more modular so it doesn't happen in the future?

Reply:
The issue here is that wagmi specifically injects the coinbase SDK connector, and when the actual extension wallet is installed, it shows both. 
So if the `coinbaseConnector` exists, then we skip the sdk one to prevent duplicates. This doesn't happen on ethers as we instantiate everything ourselves and our implementation recognizes the installation. 

Seems like we could potentially do a PR to wagmi directly as well.